src/main/java/org/wso2/carbon/module/swiftiso20022/constants/MT103Constants.java

```diff
@@ -0,0 +1,160 @@
+    public static final String ERROR_EMPTY_ENTITY_DETAIL_LINE = "Line %s of details of field %s cannot be empty";
+    public static final String ERROR_ENTITY_DETAIL_LINE_LENGTH =
+            "Line %s of details of field %s should not contain more than 35 characters ";
+    public static final String ERROR_ENTITY_DETAIL_LINE_COUNT =
+            "Details of field %s should not contain more than 4 lines";
+
+    // MT103 Text Fields
+    public static final String SENDERS_REFERENCE = "Sender's Reference";
+    public static final String TIME_INDICATION = "Time Indication";
+    public static final String BANK_OPERATION_CODE = "Bank Operation Code";
+    public static final String INSTRUCTION_CODE = "Instruction Code";
+    public static final String TRANSACTION_TYPE_CODE = "Transaction Type Code";
+    public static final String VALUE = "Value";
+    public static final String INSTRUCTED_AMOUNT = "Instructed Amount";
+    public static final String EXCHANGE_RATE = "Exchange Rate";
+    public static final String ORDERING_CUSTOMER = "Ordering Customer";
+    public static final String SENDING_INSTITUTION = "Sending Institution";
+    public static final String ORDERING_INSTITUTION = "Ordering Institution";
+    public static final String SENDERS_CORRESPONDENT = "Sender's Correspondent";
+    public static final String RECEIVERS_CORRESPONDENT = "Receiver's Correspondent";
+    public static final String THIRD_REIMBURSEMENT_INSTITUTION = "Third Reimbursement Institution";
+    public static final String INTERMEDIARY_INSTITUTION = "Intermediary Institution";
+    public static final String ACCOUNT_WITH_INSTITUTION = "Account With Institution";
+    public static final String BENEFICIARY_CUSTOMER = "Beneficiary Customer";
+    public static final String REMITTANCE_INFORMATION = "Remittance Information";
+    public static final String DETAILS_OF_CHARGES = "Details of Charges";
+    public static final String SENDERS_CHARGES = "Sender's Charges";
+    public static final String RECEIVERS_CHARGES = "Receiver's Charges";
+    public static final String SENDER_TO_RECEIVER_INFORMATION = "Sender to Receiver Information";
+    public static final String REGULATORY_REPORTING = "Regulatory Reporting";
+    public static final String ENVELOPE_CONTENTS = "Envelope Contents";
+
+    // JSON to MT103 constants
+    public static final String BLOCK01 = "block01";
+    public static final String BLOCK01_APPLICATION_ID_KEY = "applicationIdentifier";
+    public static final String BLOCK01_SERVICE_ID_KEY = "serviceIdentifier";
+    public static final String BLOCK01_LOGICAL_TERMINAL_ADDRESS_KEY = "logicalTerminalAddress";
+    public static final String BLOCK01_SESSION_NUMBER_KEY = "sessionNumber";
+    public static final String BLOCK01_SEQUENCE_NUMBER_KEY = "sequenceNumber";
+    public static final String BLOCK02 = "block02";
+    public static  final String BLOCK02_INPUT_OUTPUT_ID_KEY = "inputOutputIdentifier";
```

Comment:
shall we remove extra space from line 93-102

Reply:
Removed in commit 96d4dc498094daee16277501378ee52fbb484dd1